src/app/auth/shared/helpers/auth.guard.ts

```diff
@@ -12,6 +12,7 @@ export class AuthGuard implements CanActivate {
     ) {}
 
     canActivate(): boolean {
+        return true;
```

Comment:
Please, make sure to put a TODO comment in places like this.
Also, I already commented this guard in routes, so you can just remove this line